[9.1] support "remoteonly" value for the Access protocol parameter

alex-rg opened the pull request:
Add "remoteonly" Access -- similar to "remote" but prevents usage in local context. Can be helpful to prevent race between local/remote endpts.

BEGINRELEASENOTES
NEW: support "remoteonly" value for the Access protocol parameter -- similar to "remote" but prevents usage in local context.
ENDRELEASENOTES

chaen commented on src/DIRAC/Resources/Storage/StorageElement.py:
```diff
@@ -1153,6 +1153,10 @@ def __filterPlugins(self, methodName, protocols=None, inputProtocol=None):
+            try:
+                isRemoteOnly = pluginParameters.get("Access").lower() == "remoteonly"
+            except AttributeError:
+                isRemoteOnly = False
```
```suggestion
            isRemoteOnly = pluginParameters.get("Access","").lower() == "remoteonly"
```

chaen commented:
It seems fine to me, pending some more tests

Commit:
feat: add "remoteonly" value to the SE Access option

It's the same as "remote", but prevents using the protocol in local context (e.g. to upload data from some site's WNs to the local SE). This may be useful for sites with different storage infrastructure for local and remote transfers.
